app/models/ability.rb

```diff
@@ -187,6 +188,7 @@ def signed_in_with_cfp_role(user)
     # ids of all the conferences for which the user has the 'cfp' role
     conf_ids_for_cfp = Conference.with_role(:cfp, user).pluck(:id)
 
+    can [:index, :show, :update], Resource, conference_id: conf_ids_for_cfp
```

Comment:
@Ana06 as far as I am concerned, read is confusing, and currently our ability file does not use it anywhere (that was done on purpose, there have been comments about it in the past - which I cannot dig up though).

We also don't use update as an alias, meaning that, when we imply edit action, we write it as well. 

CanCanCan at some point had aliased different actions to `read`, and there were comments/issues about that being changed. If that happens again, I am afraid we might not catch it in time. 

I would hugely prefer that we are explicit with our abilities, rather than implicit (ie using aliases), and I strongly believe it is very important. Abilities define our authorization, if we miss something there it would mean that it gives improper access to people in our app.